frontend/packages/metal3-plugin/src/models.ts

```diff
@@ -17,7 +17,7 @@ export const NodeMaintenanceModel: K8sKind = {
   label: 'Node Maintenance',
   labelPlural: 'Node Maintenances',
   apiVersion: 'v1alpha1',
```

Comment:
Please notice that in order to go GA this should be at least v1beta1, maybe it's worth to address both in a single PR.

Reply:
Fine for me. We just have to wait if this change will get into node maintenance operator